packages/pwa-kit-create-app/scripts/create-mobify-app.js

```diff
@@ -111,7 +120,25 @@ const TEMPLATE_SOURCE_NPM = 'npm'
 const TEMPLATE_SOURCE_BUNDLE = 'bundle'
 const DEFAULT_TEMPLATE_VERSION = 'latest'
 
-const askApplicationExtensibiltyQuestions = (appExtensions) => {
+const LOCAL_DEV_PROJECT_DIR = 'dev'
```

Comment:
I reckon we have some sort of issue if the folder already exists in the extension project where we are placing the example project which lead to creating a new name "dev" over "example". I wonder if there is a way to do this in a way where the folder exists in the base project but doesn't complain when we try generating a project into it. 

We also just might want to follow the path of least resistance.